src/components/NotificationPanel/NotificationBox.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import "./NotificationBox.css";
+import PropTypes from "prop-types";
+
+const NotificationBox = ({ text, color }) => (
+  <div className="NotificationBox">
+    <div className="color_type" style={{ backgroundColor: color }} />
```

Comment:
i prefer dashes for classnames in html so `color-type`. make sure to change this in the css file too